test/bigtest/interactors/action-menu.js

```diff
@@ -3,22 +3,38 @@ import {
     property,
     clickable,
     isPresent,
-    count,
-    isVisible
-  } from '@bigtest/interactor';
+    count
+} from '@bigtest/interactor';
 
-  const ActionButtonInteractor = interactor(class ButtonInteractor {
-    clickThrough = clickable();
-  });
+const ActionButtonInteractor = interactor(class ButtonInteractor {
+  clickThrough = clickable();
+});
```

Comment:
Is this spacing adjustment intentional?

Reply:
That space was removed, in last commit doesn't exists